experimental/sql/dialects/rel_impl.py

```diff
@@ -295,26 +295,57 @@ def get(left: Operation, right: Operation,
                           result_types=[Boolean()])
 
 
-@irdl_op_definition
 class Yield(Expression):
+  """
+  Parent class of operations that gridge the gap from expressions back to
+  operators by yielding the result of an expression to the encompassing
+  operator.
+  """
+  ...
+
+
+@irdl_op_definition
+class YieldValue(Yield):
+  """
+  Bridges the gap from expressions back to operators by yielding the result of
+  an expression to the encompassing operator.
+
+  Example:
+
+  '''
+  rel_impl.yield_value(%0 : !rel_impl.bool)
+  '''
+  """
+  name = "rel_impl.yield_value"
+
+  op = OperandDef(AnyAttr())
+
+  @staticmethod
+  @builder
+  def get(op: Operation) -> 'YieldValue':
+    return YieldValue.create(operands=[op.result])
+
+
+@irdl_op_definition
+class YieldTuple(Yield):
   """
   Bridges the gap from expressions back to operators by yielding the result of
```

Comment:
Explain the concept of a tuple and how the variadic results implicitly form one?